ref/go/src/bech32/bech32_test.go

```diff
@@ -41,6 +41,17 @@ var validChecksum = []string{
 	"split1checkupstagehandshakeupstreamerranterredcaperred2y9e3w",
 }
 
+var invalidChecksum = []string{
+	" 1nwldj5",
+	"\x7F" + "1axkwrx",
```

Comment:
Why not simply `"\x7F1axkwrx"`?

Reply:
At least if the semantics are like C, that would be interpreted as hex number 0x7F1A.

Reply:
Ah, right. In C, this would give a warning (`hex escape sequence out of range`), while in Go, `\x` is followed by exactly two hex digits.

I'd change it to follow language semantics.

https://play.golang.org/p/-VJIzyF1Mf